tests/ops/qubit/test_parametric_ops.py

```diff
@@ -724,6 +726,28 @@ def test_isingzz_decomposition_broadcasted(self, tol):
 
         assert np.allclose(decomposed_matrix, op.matrix(), atol=tol, rtol=0)
 
+    def test_pc_phase_decomposition(self):
+        """ "Test that the PCPhase decomposition produces the same unitary"""
+        op = qml.PCPhase(1.23, dim=5, wires=[0, 1, 2])
+        decomp_ops = op.decomposition()
+        decomp_op = qml.prod(*decomp_ops) if len(decomp_ops) >= 1 else decomp_ops[0]
+
+        expected_mat = qml.matrix(op)
+        decomp_mat = qml.matrix(decomp_op)
+        assert np.allclose(expected_mat, decomp_mat)
```

Comment:
The decomposition result is not supposed to be `O_1 O_2 \dots O_n`?

Reply:
I don't understand the question?

Reply:
Minor point, I was curios why the actual decomposition ops are not compared instead of their matrix.

Reply:
It's quite a long decomposition, I can compute it by hand for one case. My thought process was that ultimately a decomposition just needs to produce the same matrix representation as the operator itself. So I just checked that.

Reply:
Sounds good.